citystories.json

```diff
@@ -0,0 +1,163 @@
+{
```

Comment:
I don't think this file was meant to be included

Reply:
Was removed in the latest commit